app/web_framework/cluster_master_application.py

```diff
@@ -418,3 +422,16 @@ def post(self):
             [int(slave_id) for slave_id in self.decoded_body.get('slaves')]
 
         self._cluster_master.set_shutdown_mode_on_slaves(slaves_to_shutdown)
+
+
+class _SlavesHeartbeatHandler(_ClusterMasterBaseAPIHandler):
+    def get(self):
```

Comment:
The idea was to add endpoint for heartbeat info for all the slaves, but I think it can be cleaned up since it is now exactly same as GET on `/slave` endpoint